src/ProjNavigator/tcl_command_integration.cpp

```diff
@@ -41,10 +41,28 @@ bool TclCommandIntegration::TclSetTopModule(int argc, const char *argv[],
     return false;
   }
 
-  const QString topModule = QString(argv[1]);
+  auto topModule = QString{};
```

Comment:
this could be done more simple. Like:

  const QString topModule = QString(argv[1]);
  QString topModuleLib{};
  if (argc > 3) {
    if (QString{argv[2]} != "-work") {
      out << "Incorrect syntax .....";
      return false;
    }
    topModuleLib = QString(argv[3]);
  }
  m_projManager->setCurrentFileSet(m_projManager->getDesignActiveFileSet());
  int ret = m_projManager->setTopModule(topModule);
  m_projManager->setTopModuleLibrary(topModuleLib);
  if (0 == ret) update();
  return true;